web/webkit/src/main/scala/net/liftweb/http/LiftSession.scala

```diff
@@ -2327,13 +2327,13 @@ class LiftSession(private[http] val _contextPath: String, val underlyingId: Stri
    * the CometActor is instantiated.  If the CometActor already exists
    * in the session, the message will be sent immediately.  If the CometActor
    * is not yet instantiated, the message will be sent to the CometActor
-   * as part of setup (@see queueCometMessage) if it is created as part
+   * as part of setup (see queueCometMessage) if it is created as part
```

Comment:
Was suggesting this be \<backtick\>[[queueCometMessage]]\<backtick\>, so that the generated scaladocs (a) style `queueCometMessage` as monospace and (b) link to the other one.